samples/bot-conversation/csharp/AppManifest/manifest.json

```diff
@@ -68,5 +68,19 @@
     "identity",
     "messageTeamMembers"
```

Comment:
Please change updated date for sample.json inside assets folder

Reply:
Fixed